build.sbt

```diff
@@ -618,4 +620,7 @@ def CiCommand(name: String)(commands: List[String]): Command = Command.command(n
   }
 }
 def ci(command: String) = s"plz $ciScalaVersion $command"
+lazy val scala210 = "2.10.6"
+lazy val scala211 = "2.11.11"
+lazy val scala212 = "2.12.2"
```

Comment:
Allright, an alternative suggestion. Have file called `Versions.scala` in `build/`. And then call these guys `LatestScala210`, `LatestScala211` and `LatestScala212`.